src/kili/cli/project/member/helpers.py

```diff
@@ -49,31 +53,32 @@ def collect_members_from_csv(csv_path: str, role: Optional[str]):
 
 def collect_members_from_project(kili, project_id_source: str, role: Optional[str]):
     """copy members from project of id project_id_source"""
-    members = []
+    activated_members = []
 
     if role is not None:
         raise ValueError("--role cannot be used if the argument passed is a Kili project_id")
 
     try:
-        users = cast(
+        existing_members = cast(
             List[Dict],
-            kili.project_users(
-                project_id=project_id_source,
+            ProjectUserQuery(kili.auth.client)(
+                where=ProjectUserWhere(project_id=project_id_source),
                 fields=["role", "user.email", "activated"],
-                disable_tqdm=True,
             ),
         )
-        for user in users:
-            if user["activated"]:
-                members.append({"email": user["user"]["email"], "role": user["role"]})
+        for existing_member in existing_members:
+            if existing_member["activated"]:
+                activated_members.append(
+                    {"email": existing_member["user"]["email"], "role": existing_member["role"]}
+                )
     except:
```

Comment:
we don't have a specific error for this except?

Reply:
i tested, it in fact does not fail in the try when the project does not exist... I will change the error raised